shared/app/nav.native.js

```diff
@@ -198,6 +198,8 @@ class MainNavStack extends Component<any, any> {
     const stacks = this._stackCache
       .set(props.routeSelected, props.routeStack)
       .toArray()
+      // The inner views are replaced with placeholders so lets not even both to render the siblings
+      .filter(([key, stack]) => key === props.routeSelected)
```

Comment:
so a long long long time ago we kept tab screens alive if you switched away. Then later we started replacing some inner inner inner child with just an empty view. We might as well not even render the parents of this view. I've noticed no visible changes with this change but it'll reduce the view hierarchy thats alive